src/parsing/medium.py

```diff
@@ -304,8 +304,8 @@ async def validate(self, flush: bool = False, reason: Union[Exception, str] = No
                     if url.startswith(env.IMAGES_WESERV_NL) or url.startswith(env.IMG_RELAY_SERVER):
                         continue
                     medium_info = await web.get_medium_info(env.IMG_RELAY_SERVER + url)
-                    if medium_info is None:
-                        continue
+                if medium_info is None:
+                    continue
```

Comment:
If `medium_info is None` is `false` at the previous condition, this change will introduce a new redundant condition judgment.